pkg/ipcache/metadata.go

```diff
@@ -337,13 +336,13 @@ func (m *metadata) findAffectedChildPrefixes(parent cmtypes.PrefixCluster) (chil
 // the caller must trigger forced policy regenerations on all endpoints.
 //
 // Do not call this directly; rather, use TriggerLabelInjection()
-func (ipc *IPCache) doInjectLabels(ctx context.Context, modifiedPrefixes []cmtypes.PrefixCluster) (remainingPrefixes []cmtypes.PrefixCluster, mutated bool, err error) {
+func (ipc *IPCache) doInjectLabels(ctx context.Context, modifiedPrefixes []cmtypes.PrefixCluster) (remainingPrefixes []cmtypes.PrefixCluster, err error) {
```

Comment:
nit: The doc comment here still mentions the removed `mutated` return value. Might be worth documenting that this function now also triggers policy regeneration.